### DISCUSSION TITLE
BlenderBot2.0

### DISCUSSION THREAD

**Discussion by jh-skaion** [facebookresearch/ParlAI#3832] [Q&A]
Hello team,
i was looking into the blenderbot2.0 and i decided to try it but i am getting error when trying to run..
![blenderbot20](https://user-images.githubusercontent.com/82824999/126556267-6fad95a7-823d-4cf7-881d-4b3017ac0292.png)

i created a conda env call ParlAI and installed the parlai 1.3.0 version. i downloaded the parlai git to my documents folder. i copied the zoo blenderbot2.0 to the anaconda3 parlai install folder. i copied the folder project  too but i still getting the error.

please can you help, thank you

**Comment by jh-skaion** [comment]
i also got an error for Wizint Fid Search Query Search Engine
![WizInt](https://user-images.githubusercontent.com/82824999/126562136-17b2dd55-d232-4b4c-b7e7-acfbfa3881a7.png)

**Comment by klshuster** [comment]
Hi there, the code for this project has not made it into a release yet (we plan on making a new release soon). If you'd like to use the code before then, you can install ParlAI via the "Development Installation" route described here: https://github.com/facebookresearch/ParlAI#installing-parlai

**Reply by jh-skaion** [reply]
Thanks for the info

**Reply by klshuster** [reply]
FYI, code and models for BlenderBot 2.0 are now in ParlAI's latest release (1.4.0)

**Reply by jh-skaion** [reply]
oh great thank you @klshuster...

**Comment by jh-skaion** [comment]
hello @klshuster 
i am getting an error running the bb2
![image](https://user-images.githubusercontent.com/82824999/128437222-b6862e45-fbac-4382-8618-b3c044bf1326.png)

can you assist?

**Reply by klshuster** [reply]
please upgrade to version 1.4.1
